htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -774,7 +779,15 @@ public function getNomUrl($withpicto = 0, $option = '', $notooltip = 0, $save_la
 		if ($user->rights->fournisseur->commande->lire) {
 			$label = '<u class="paddingrightonly">'.$langs->trans("SupplierOrder").'</u>';
 			if (isset($this->statut)) {
-				$label .= ' '.$this->getLibStatut(5);
+				$statusText = ' '.$this->getLibStatut(5);
+				$parameters = array('obj' => $this);
+				$reshook = $hookmanager->executeHooks('moreHtmlStatus', $parameters, $object); // Note that $action and $object may have been modified by hook
```

Comment:
Because a hooks is already inside LibStatut(), there is no need to add another one here.